src/store/addressbooks.js

```diff
@@ -553,17 +553,7 @@ const actions = {
 		const vData = contact.vCard.toString().replace(/^UID.+/im, '')
 		const newContact = new Contact(vData, addressbook)
 
-		try {
```

Comment:
You're not syncing the contact to the server anymore? :thinking:

Reply:
It syncs just fine, once you've edited the contact. I believe it is added to the local store, and once edited, is then synced.

It behaves as if you have created a new contact, but with the fields pre-filled. This behaviour is intended, since a direct clone of the contact, since in most cases you would want to change a piece of information before saving.

Reply:
> once you've edited the contact

This is the issue, this is not the initial goal of that UX.

Reply:
Okay, so it needs to be reworked to sync the contact to the server before updating the context?